launch/launch/utilities/signal_management.py

```diff
@@ -143,6 +142,9 @@ def __on_sigterm(signum, frame):
             # Windows does not support SIGQUIT
             signal.signal(signal.SIGQUIT, __on_sigquit)
     except ValueError:
-        _logger.error("failed to set signal handlers in 'launch.utilities.signal_management.py'")
-        _logger.error('this function must be called in the main thread')
+        logger = logging.getLogger(__name__)
```

Comment:
Is this the same as it was before? (`launch.utilities.signal_management`)

Reply:
So, `__name__` seems to be populated w.r.t. the `__main__` script. A script would normally include `launch`, that may in turn include `.utilities.signal_management` and thus the full name will `launch.utilities.signal_management`.

Reply:
My main concern is that if launch is embedded into another Python project or something like that, then `__name__` might change, making the logger name unpredictable. I have a slight preference for the hard coded string over `__name__`, but I'll leave it up to you. I don't consider this blocking.

Reply:
Hmm, that's true. Though it's not strictly unpredictable, if one embeds `launch` in a e.g. `hate_pypi` package, the logger will be named after the module the action is in. That changed. Because `launch` became a module in a package.

------

Honestly, I'd like to provide a hint as to which action _instance_ is actually logging what instead. But for a first shot I think I'm happy with this.